Joystream/joystream
# Pioneer: Apps adjustments + joy-members reactivation

**Implements:**
- https://github.com/Joystream/joystream/issues/1131
- https://github.com/Joystream/joystream/issues/1132

**Changes:**
- Adjusted styles and functionality of the new `polkadot-js/apps` based on our old, modified apps (ie. added providers in `Apps`, added our types to api, added `TopBar` with account dropdown-select, adjusted the routes/tabs). I tried to limit the amout of those changes to the absolute minimum.
- Partially re-activated `joy-utils`. Moved all required functionality to new `joy-utils` directory, did some refactoring and put the files inside a new, more clean directory structure.
- Reactivated `joy-members` and `joy-pages` and did all the necessary adjustments.
- Disabled some linter rules that could cause too much chaos during the upgrade 
- Fixed remaining linter issues

**Draft:**
Based on: https://github.com/Joystream/joystream/pull/1101
Merged https://github.com/Joystream/joystream/pull/1153 for new node compatibility

Commits of intrest:
- Almost all of the work: https://github.com/Joystream/joystream/commit/ea711c9e7731237bc987aa2e0f493d41488fb907
- Linter rules adjustment + autofix: https://github.com/Joystream/joystream/commit/15e1c69e5764f2b03f0a4eec6ce92560034f95ce
- Manual linter fixes: https://github.com/Joystream/joystream/commit/83d22c4016c293500857ce1093f65d0d31fcfcac
- Icons/title changes: https://github.com/Joystream/joystream/commit/8a8f85aee35153a905856dcfb4ab4b8bd4e97f11

**Testing:**
1. Run dev chain based on https://github.com/Joystream/joystream/pull/1126/commits/fb9845beb7773e2282057572488259bf32487218 (currently should already be part of https://github.com/Joystream/joystream/tree/substrate_version_upgrade)
1. `yarn workspace pioneer start` or use deploy preview at https://deploy-preview-1156--joystream-pioneer.netlify.app/
1. Change endpoint to local node (ie. by clicking the Joystream logo in the top-left corner)
